### PR TITLE
Add resize event to splitter

### PR DESCRIPTION
**Pull request by Ice-Hazymoon**
Add resizing event to splitter component

```html
<Splitter style="height: 300px" class="mb-5" @resizing="console.log">
    <SplitterPanel class="flex align-items-center justify-content-center"> Panel 1 </SplitterPanel>
    <SplitterPanel class="flex align-items-center justify-content-center"> Panel 2 </SplitterPanel>
</Splitter>
```

### CODE REVIEW

**Comment by mertsincan** [comment]
Thanks a lot for your contribution! I think its name can be 'resize' instead of 'resizing'. Also, could you please create a Github issue and link it to this PR? Then, I'll merge it asap.

File: components/lib/splitter/Splitter.d.ts [hunk 1]
```diff
@@ -185,6 +200,11 @@ export interface SplitterEmits {
      * @param {SplitterResizeStartEvent} event - Custom resize start event.
      */
     resizestart(event: SplitterResizeStartEvent): void;
+    /**
+     * Callback to invoke when resizing.
+     * @param {SplitterResizeEvent} event - Custom resizing event.
+     */
+    resizing(event: SplitterResizeEvent): void;
```
Comment (mertsincan) [inline components/lib/splitter/Splitter.d.ts:207]: Can you change this line? Thanks a lot!

**Comment by mertsincan** [comment]
Thanks a lot! Also, please see https://github.com/primefaces/primevue/pull/4526/files#r1347812333

**Comment by Ice-Hazymoon** [comment]
> Thanks a lot! Also, please see https://github.com/primefaces/primevue/pull/4526/files#r1347812333

Sorry, I fixed it.

**Comment by mertsincan** [comment]
Thanks a lot for your contribution!

**Pull request merged**